enums.go

```diff
@@ -0,0 +1,282 @@
+package openrtb
+
+//enums and constants for tables 5.x.x (except 5.1 - IAB categories)
+
+//5.2 Banner Ad Types
+
+const (
+	EBannerAdTypeXHTMLTextAd int8 = iota + 1
```

Comment:
Better not use this for OpenRTB enums - they're already defined in spec, don't rely on incremental `iota`.

Reply:
Also, no need for `E` prefix. It won't really help you with autocompletion, so effectively it's useless.

Also, I haven't seen Hungarian notation in Go std lib.

And, as I remember, enum names don't really clash with object names, so really - no need for prefixing.